src/multiscan_driver.cpp

```diff
@@ -159,6 +163,10 @@ MultiscanNode::MultiscanNode(bool autostart) :
     this->metrics.num_threads_pub = this->create_publisher<std_msgs::msg::UInt32>(
                                                         "multiscan_driver/process_metrics/num_threads",
                                                         rclcpp::SensorDataQoS{} );
```

Comment:
It would be nice if you could use the HAS_SENSORS macro around this so this only happens when we need it